bundler/lib/bundler/cli/common.rb

```diff
@@ -14,6 +14,19 @@ def self.print_post_install_message(name, msg)
       Bundler.ui.info msg
     end
 
+    def self.output_fund_metadata_summary
+      current_dependencies = Bundler.definition.requested_dependencies.map(&:name)
```

Comment:
Another nit, but for simmetry, could we avoid the `map` here, and instead do `dep.name` inside the `count` block?

Reply:
Also, I'm wondering whether we should actually use `requested_dependencies` here, since it kind of makes the message inaccurate because of taking groups into account. So, if your current environment is using, say, `without production`, it will omit to include the `production` gems in the count. But technically you still depend on those gems (they're pretty critical in this case, actually :sweat_smile:).

Reply:
> Also, I'm wondering whether we should actually use `requested_dependencies` here, since it kind of makes the message inaccurate because of taking groups into account.

I think I agree with you, and made that change!